docs/src/main/tut/typeclasses/nonemptytraverse.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: docs
+title:  "NonEmptyTraverse"
+section: "typeclasses"
+source: "core/src/main/scala/cats/NonEmptyTraverse.scala"
+scaladoc: "#cats.NonEmptyTraverse"
+---
+
+# NonEmptyTraverse`
+
+`NonEmptyTraverse` is a non-empty version of the `Traverse` type class, just like `Reducible` is a non-empty version of `Foldable`.
```

Comment:
Should we link to the [`Traverse`](https://typelevel.org/cats/typeclasses/traverse.html) documentation here?

Reply:
Yes, fixed :)